designsafe/static/scripts/data-depot/components/projects/pipeline-publish/pipeline-agreement.html

```diff
@@ -46,4 +46,20 @@ <h4 class="modal-title" style="border:none;">
                 </button>
         </div>
     </div>
+    <div data-ng-if="$ctrl.existingPub">
+        <div class="alert alert-warning">
+            This project has already been published.
+            If you would like to update your existing publication or
+            if you are having trouble viewing your publication please
+            <a href="/help/new-ticket/" target="_blank">submit a ticket</a>.
```

Comment:
consider adding some prefilled ticket info here
```suggestion
            <a href="/help/new-ticket/?category=DATA_CURATION_PUBLICATION&subject=Update+existing+publication" target="_blank">submit a ticket</a>.
```

Reply:
Good Idea! I'll add that in there.